### PR TITLE
Fixing anonymous user image is not showing up

### LINKED ISSUE #736: Anonymous user image is not showing up
http://bugheist.com/issue/1520

### PR DESCRIPTION
**Pull request by spiderxm**
Screenshot 2021-06-01 at 8 51 31 PM

Fixing issue #736

### CODE REVIEW

**Comment by spiderxm** [comment]
@fredfalcon i think we should not lint HTML files. It is linting them on basis of normal HTML files rather than Django templates.

File: website/templates/_activity.html [hunk 1]
```diff
@@ -4,9 +4,13 @@
...
             <img src="{{ activity.user.userprofile.avatar }}" width="100" class="img-responsive img-rounded">
         {% elif activity.user.socialaccount_set.all.0.get_avatar_url %}
-            <img src="{{ activity.user.socialaccount_set.all.0.get_avatar_url }}" width="100" class="img-responsive img-rounded">
-        {% else %}
+            <img src="{{ activity.user.socialaccount_set.all.0.get_avatar_url }}" width="100"
+                 class="img-responsive img-rounded">
+        {% elif activity.user.email %}
             <img src="{% gravatar_url activity.user.email 100 %}" width="100" class="img-responsive img-rounded">
+        {% else %}
+            <img src="https://heightstitle.com/wp-content/uploads/2014/12/user-dummy-pic.png" width="100"
```
Comment (fredfalcon) [inline website/templates/_activity.html:12]: @spiderxm can you please move this image into static?  The first time I tried to load it, I got a 404, so will be good to have it hosted with us.

**Review by fredfalcon — CHANGES_REQUESTED** [review]

File: website/templates/_activity.html (hunk 1 above)
Comment (spiderxm) [reply website/templates/_activity.html:12]: ok sure

**Review by fredfalcon — APPROVED** [review]

**Comment by fredfalcon** [comment]
Thank you!

**Pull request merged**